back/engines/commercial/bulk_import_ideas/app/services/bulk_import_ideas/import_project_ideas_service.rb

```diff
@@ -6,7 +6,7 @@ def initialize(current_user, project_id, locale, phase_id)
       super(current_user)
       @project = Project.find(project_id)
       @phase = phase_id ? @project.phases.find(phase_id) : TimelineService.new.current_phase(@project)
-      @form_fields = IdeaCustomFieldsService.new(Factory.instance.participation_method_for(@phase || @project).custom_form).enabled_fields
+      @form_fields = IdeaCustomFieldsService.new(Factory.instance.participation_method_for(@phase || @project).custom_form).submittable_fields
```

Comment:
@luucvanderzee instead of having the array of question types in the form parser the custom fields service has an array of submittable fields already defined.

Reply:
Ok but it seems like they are slightly different. The `idea_plaintext_parser` cannot deal with linear scales (and in your branch, also not with number inputs yet). And we also need to remove the tags, image upload fields etc

Reply:
Good point, they're also not needed in the rest of the code, so I will add that exemption where we define the fields